mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/AccountsScreen.kt

```diff
@@ -42,11 +47,26 @@ fun AccountsScreen(
 ) {
     val accountsUiState by viewModel.accountsUiState.collectAsStateWithLifecycle()
     val sampleList = viewModel.bankAccountDetailsList
-    AccountScreen(
-        accountsUiState = accountsUiState,
-        onAddAccount = onAddAccount,
-        sampleList = sampleList,
-    )
+    var isRefreshing by rememberSaveable {
+        mutableStateOf(false)
+    }
+    val swipeRefreshState = rememberSwipeRefreshState(
+            isRefreshing = isRefreshing
+        )
+        SwipeRefresh(
+            state = swipeRefreshState,
+            onRefresh = {
+                isRefreshing = true
+                viewModel.fetchLinkedAccount()
+                isRefreshing = false
+            }
```

Comment:
we can create a function `fun refreshAccountList()` and write this logic there and consequently use this function in the composable. Replicate this in all of the screens with appropriate function names